src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -47,8 +46,9 @@ byte_size.defaultOptions({
 
 const MY_CONNECTION_USER_ID = '0';
 
-Polymer({
-  _template: html`
+export class ServerView extends DirMixin(PolymerElement) {
```

Comment:
nit: should it be `OutlineServerView` for consistency with the file name?

Reply:
I was thinking no. The reason why we have `outline-` is namespacing. All elements are globals. With a module system, we don't need to add scope in the names themselves.